Loop.xcodeproj/project.pbxproj

```diff
@@ -2636,7 +3160,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer: loudnate@gmail.com (XZN842LDLT)";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 56;
+				CURRENT_PROJECT_VERSION = 55;
```

Comment:
Did the project version go down?

Reply:
This is due to a wrinkle in the space-time continuum. I'll bump it.